{
  throw new IOException("Simulated unexpected error.");
}
